src/libs/ErrorUtils.js

```diff
@@ -88,6 +88,27 @@ function getLatestErrorField(onyxData, fieldName) {
         .value();
 }
 
+/**
+ * Get the first error in the errorFields ordering by its key
```

Comment:
We don't need the comment because the function name already explains what it is.